### PR TITLE
Support extra custom config for prometheus

### PR DESCRIPTION
**Pull request by tuteng**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

Check the box below.

Need to update docs? 

- [x] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [ ] `no-need-doc` 
  
  (Please explain why)
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: charts/sn-platform-slim/aa.log
Comment (FushuWang) [inline charts/sn-platform-slim/aa.log:1]: This file is not suppose to be added

File: charts/sn-platform-slim/tt.yaml
Comment (FushuWang) [inline charts/sn-platform-slim/tt.yaml:1]: This file is not suppose to be added

File: charts/sn-platform/templates/prometheus/prometheus-configmap.yaml [hunk 1]
```diff
@@ -155,6 +155,11 @@ data:
 {{- with .Values.prometheus.extraScrapeConfigs }}
 {{ toYaml . | indent 4 }}
 {{- end }}
+{{- end }}
+{{- if .Values.prometheus.extraRemoteWrite -}}
```
Comment (FushuWang) [inline charts/sn-platform/templates/prometheus/prometheus-configmap.yaml:159]: The configuration item key `remote_write` should be added, and it is at the same level as the key`global`

File: charts/sn-platform-slim/tt.yaml
Comment (tuteng) [reply charts/sn-platform-slim/tt.yaml:1]: Thanks

File: charts/sn-platform/templates/prometheus/prometheus-configmap.yaml (hunk 1 above)
Comment (tuteng) [reply charts/sn-platform/templates/prometheus/prometheus-configmap.yaml]: Yes, this is generated config file

```
  prometheus.yml: |
    global:
      scrape_interval: 15s
    rule_files:
      - 'rules.yml'
    alerting:
      alertmanagers:
      - static_configs:
        - targets: ['pulsar19-sn-platform-alert-manager:9093']
        path_prefix: /
    scrape_configs:
    - job_name: 'prometheus'
      static_configs:
      - targets:
        - '127.0.0.1:9090'
      metrics_path: /metrics
    - job_name: 'kubernetes-pods'
      bearer_token_file: /pulsar/tokens/client/token
      kubernetes_sd_configs:
      - role: pod
      relabel_configs:
      - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
        action: keep
        regex: true
      - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_path]
        action: replace
        target_label: __metrics_path__
        regex: (.+)
      - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
        action: replace
        regex: ([^:]+)(?::\d+)?;(\d+)
        replacement: $1:$2
        target_label: __address__
      - action: labelmap
        regex: __meta_kubernetes_pod_label_(.+)
      - source_labels: [__meta_kubernetes_namespace]
        action: replace
        target_label: kubernetes_namespace
      - source_labels: [__meta_kubernetes_pod_label_component]
        action: replace
        target_label: job
      - source_labels: [__meta_kubernetes_pod_name]
        action: replace
        target_label: kubernetes_pod_name
      metric_relabel_configs:
    - job_name: 'kubernetes-nodes'
      scheme: https
      kubernetes_sd_configs:
        - role: node

      tls_config:
        ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
      bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token

      relabel_configs:
        - action: labelmap
          regex: __meta_kubernetes_node_label_(.+)
        - target_label: __address__
          replacement: kubernetes.default.svc:443
        - source_labels: [__meta_kubernetes_node_name]
          regex: (.+)
          target_label: __metrics_path__
          replacement: /api/v1/nodes/${1}/proxy/metrics
    - job_name: 'kubernetes-cadvisor'
      scheme: https
      kubernetes_sd_configs:
        - role: node

      tls_config:
        ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
      bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token

      relabel_configs:
        - action: labelmap
          regex: __meta_kubernetes_node_label_(.+)
        - target_label: __address__
          replacement: kubernetes.default.svc:443
        - source_labels: [__meta_kubernetes_node_name]
          regex: (.+)
          target_label: __metrics_path__
          replacement: /api/v1/nodes/${1}/proxy/metrics/cadvisor
    remote_write:
    - basic_auth:
        password: secret-password
        username: admin
      url: http://prometheus-service.prometheus.svc.cluster.local:9090/api/v1/write
```

**Review by FushuWang — APPROVED** [review]

**Pull request merged**